tools/build/ms-plugins.coffee

```diff
@@ -0,0 +1,113 @@
+{ walkFiles } = require('@resin.io/doxx-utils')
+HbHelper = require('@resin.io/doxx-handlebars-helper')
+
+path = require('path')
+fse = require('fs-extra')
+semver = require('semver')
+
+utils = require('../lib/utils')
+pythonHelper = require('../lib/python')
+
+
+removeFile = (obj, file, files) ->
+	if obj and obj is utils.UNSUPPORTED
+		delete files[file]
+
+copyFile = (src, dest) ->
+	try
+		fse.copySync(src, dest)
+	catch err
+		console.error(err)
+
+copyDebianFile = (obj, destDir) ->
+	copyFile('./supporting_files/debian.01_buildconfig', "#{destDir}/01_buildconfig")
+	copyFile('./supporting_files/debian.01_nodoc', "#{destDir}/01_nodoc")
+
+	# Systemd
+	if obj.$debian_suite.id is 'wheezy'
+		copyFile('./supporting_files/debian.entry-nosystemd.sh', "#{destDir}/entry.sh")
+	else
+		copyFile('./supporting_files/debian.entry.sh', "#{destDir}/entry.sh")
+		copyFile('./supporting_files/debian.launch.service', "#{destDir}/launch.service")
+
+	# QEMU
+	if obj.$debian_arch.id is 'aarch64'
```

Comment:
The approach suggested above doesn't work for binaries though, but I'd arrange this specific copying in the shell script after the entire tree is built.